src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/src/upd_request_operation.c

```diff
@@ -511,7 +515,7 @@ static void HandleReportResultsAck(ADUC_Retriable_Operation_Context* context, AD
                 context->lastSuccessTime = nowTime;
                 context->nextExecutionTime = nowTime + 60; // TODO: add random jitter and use retry configuration.
                 ADUC_StateStore_SetReportResultsAck(false);
-                AduUpdUtils_TransitionState(ADU_UPD_STATE_IDLEWAIT, updData);
+                AduUpdUtils_TransitionState(ADU_UPD_STATE_IDLEWAIT, updData, context);
```

Comment:
Do we have a bug or anything else to track the todos for adding random jitter and using the retry configuration?

Reply:
This is tracking fixing all of these holistically (things outside the scope of this PR): [Bug 48124671](https://microsoft.visualstudio.com/OS/_workitems/edit/48124671): [gen2] Simplify timeout/retry config settings and use them instead of hard-coded defaults.